cli/sawtooth_cli/submit.py

```diff
@@ -78,7 +78,7 @@ def do_submit(args):
 
         print('Wait timed out! Some batches have not yet been committed...')
         for batch_id, status in statuses.items():
-            print('{:128.128}  {:8.8}'.format(batch_id, status))
+            print('{:128.128}  {:10.10}'.format(batch_id, status))
```

Comment:
It's a pattern used elsewhere in the CLI, printing out data in a fixed size table including when the data has variable print lengths. In this case it's the second of just two columns, and the first column should always be 128 character unless something weird happens, so the pattern is less applicable, but I think `{10:10}` is fine.